tests/Nox.Types.Tests/Types/DateTimeDuration/DateTimeDurationTests.cs

```diff
@@ -0,0 +1,350 @@
+﻿using FluentAssertions;
+
+using System.Globalization;
+
+namespace Nox.Types.Tests.Types;
+
+public class DateTimeDurationTests
+{
+    [Fact]
+    public void From_WithTimeSpanInput_ReturnsValue()
+    {
+        var timeSpan = new TimeSpan(1, 2, 3, 4, 5);
+
+        var dateTimeDuration = DateTimeDuration.From(timeSpan);
+
+        dateTimeDuration.Value.Should().Be(timeSpan);
+    }
+
+    [Fact]
+    public void From_WithNegativeTimeSpanInput_ReturnsAbsoluteValue()
+    {
+        var timeSpan = new TimeSpan(1, 2, 3, 4, 5);
+
+        var dateTimeDuration = DateTimeDuration.From(-timeSpan);
+
+        dateTimeDuration.Value.Should().Be(timeSpan.Duration());
```

Comment:
Can we have a hardcoded expected value instead of timeSpan.Duration()?

Reply:
Thanks for the suggestion, I've updated the tests